FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,55 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard self != nil else { return }
+
+			switch result {
+			case let .success((data, httpResponse)):
+				guard httpResponse.statusCode == 200 else {
+					completion(.failure(Error.invalidData))
+					return
+				}
+
+				do {
+					let decoder = JSONDecoder()
+					let feedImages = try decoder.decode(FeedImageResponse.self, from: data)
+						.items
+						.map { $0.feedImage }
+
+					completion(.success(feedImages))
+				} catch {
+					completion(.failure(Error.invalidData))
+				}
+
+			case .failure:
+				completion(.failure(Error.connectivity))
+			}
+		}
+	}
+
+	private struct FeedImageResponse: Codable {
+		let items: [FeedImageRepresentation]
+	}
+
+	private struct FeedImageRepresentation: Codable {
```

Comment:
Please see my previous comment regarding `Codable` conformance.
```suggestion
	private struct FeedImageRepresentation: Decodable {
```